ckan/views/group.py

```diff
@@ -142,7 +115,8 @@ def index(group_type: str, is_organization: bool) -> str:
     }
 
     try:
-        assert _check_access(u'group_list', context)
+        action_name = 'organization_list' if is_organization else 'group_list'
+        assert check_access(action_name, context)
```

Comment:
seems odd that we assert `check_access` since this function either returns True or raises a NotAuthorized, right?